packages/dlog/src/dlog.ts

```diff
@@ -111,10 +109,10 @@ const _cloneAndFormat = (
                 }
             }
         }
-        return newObj
+        return JSON.stringify(newObj)
     }
 
-    return obj
+    return JSON.stringify(obj)
```

Comment:
i don't think we want to json things here. this will be really slow.
is this the only way?